admin/src/templates/edit_translation.html

```diff
@@ -10,7 +10,14 @@
   <form action="" method="POST">
     <div class="form-group">
       <label translate>translation.key</label>
-      <input class="form-control" ng-model="model.key" ng-disabled="editing">
+      <div class="col-sm-11 no-padding-horizontal">
+        <input class="form-control" ng-model="model.key" ng-disabled="editing">
+      </div>
+      <div class="col-sm-1 no-padding-horizontal">
+        <button type="button" class="btn btn-link btn-danger" ng-click="delete()">
+            <i class="fa fa-trash-o"></i>
+          </button>
+      </div>
```

Comment:
I think it would make more sense to have the delete button in the footer of the modal rather than in the key section because you're deleting the key and all the values.

I'm thinking it should be a `btn-danger` (no `btn-link`) with a trash can and the word "delete" left aligned in the footer.

Also it should only be displayed when editing and hidden when adding a new key.

What do you think?

Reply:
Agreed ...

<img width="642" alt="screen shot 2018-11-27 at 10 26 21 am" src="https://user-images.githubusercontent.com/1445164/49092137-50560700-f22f-11e8-86e3-da48f4e940db.png">

Reply:
Agreed ...

<img width="625" alt="screen shot 2018-11-27 at 10 35 27 am" src="https://user-images.githubusercontent.com/1445164/49092509-34069a00-f230-11e8-96e3-c8def0fd7038.png">